electron/main/Files/Types.ts

```diff
@@ -18,7 +18,7 @@ export const isFileNodeDirectory = (fileInfo: FileInfoNode): boolean => {
 
 export interface AugmentPromptWithFileProps {
   prompt: string;
-  llmSessionID: string;
+  llmName: string;
```

Comment:
Yes which is nice because we can set a time for ollama to automatically unload a model if it has been loaded into memory. This defaults to 5 minutes I think